src/core/changes-since.js

```diff
@@ -0,0 +1,82 @@
+// @ts-check
+/**
+ * Fetches commits since a given commitish (tag or commit) and shows them as a
+ * list of "changes since" given commitish. If a filter function is provided by
+ * the user, it is used to filter the commits that are to be shown. Otherwise,
+ * all commits are shown.
+ */
+import { pub } from "./pubsubhub.js";
+
+export const name = "core/changes-since";
+
+export async function run(conf) {
+  const { changesSince } = conf;
+  if (!changesSince) return; // nothing to do
+
+  const el = document.querySelector("ul#changes-since");
```

Comment:
I kinda like this, because it sidesteps the l10n issues and us having to add a heading.... but at the same time, it's different to most of the other ones, which rely on `<section id=>`... I guess in a way this is better because it gives more flexibility.

Reply:
OMG OMG OMG!  wait!!!!! what if we don't have a configuration option at all! But instead:

```HTML 
<ul id='changes-since' data-from="commit" data-to="tag"></ul>
```


That would be awesome! So, imagine:

```
### Changes between FPWD and CR
<ul id='changes-since' data-from="FPWD" data-to="CR"></ul>

### Changes since FPWD
<ul id='changes-since' data-from="cava2fdasdfawsd3" data-to="FPWD"></ul>
```

🤩🤩🤩🤩🤩🤩

Reply:
Made some additional changes above.

Reply:
One more... what if we make this out first custom element? 

```HTML
<rs-changelog from="" to="" filter=""></rs-changelog>
```

Reply:
I did consider this approach. Wasn't sure as the filter function would need to be registered globally, whereas with respecConfig, it can be a block level or even inline. Might be a non issue.

Reply:
Also, the `to` field is presently not supported on respec.org. Do we want to support it?

Reply:
> I did consider this approach.

I don't doubt it! 

> Might be a non issue.

I think it's a non-issue. 

> Also, the to field is presently not supported on respec.org. Do we want to support it?

It be great if we could... doing a range would be amazing.